compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Recursive.hs

```diff
@@ -177,7 +179,7 @@ instance Recursive Expr where
     ETyLam      a b   -> ETyLamF        a b
     ENil        a     -> ENilF          a
     ECons       a b c -> EConsF         a b c
-    ECase       a b   -> ECaseF         a (map projectCaseAlternative b)
+    ECase       a b   -> ECaseF         a (Prelude.map projectCaseAlternative b)
```

Comment:
What’s up with that? You imported `Text` qualified (and it should be) so I don’t see why this needs to be qualified.